README.md

```diff
@@ -1,55 +1,30 @@
 # Elixir Koans
 
-[![Build Status](https://travis-ci.org/elixirkoans/elixir-koans.svg?branch=master)](https://travis-ci.org/elixirkoans/elixir-koans)
+[![Build Status](https://travis-ci.org/craigjbass/elixir-koans.svg?branch=master)](https://travis-ci.org/craigjbass/elixir-koans)
```

Comment:
This just seemed logical to me unless we are going to try and get the changes merged upstream.

Reply:
Need to convert this to GitHub actions, as travis no longer offers free tier for OS